### DISCUSSION TITLE
InfillPattern from Layer Settings block

### DISCUSSION THREAD

**Discussion by Ngm0505** [HALRobotics/Beta#99] [Additive Manufacturing Processes]
Greetings, 
Does anyone knows how to specify the infill pattern from Layer Settings block?
Thanks in advance!
![img](https://user-images.githubusercontent.com/67653786/117669996-c94f3c00-b1a7-11eb-8633-8b305b6ba057.png)

**Comment by Thomas-Delaplanche** [comment]
Hi, you need to use the component "Pattern Settings". Note that by right-clicking on "Type" you can choose between different patterns. 
![image](https://user-images.githubusercontent.com/60965104/117682652-9317b980-b1b3-11eb-9e73-94cf48d5d3c7.png)
![image](https://user-images.githubusercontent.com/60965104/117683195-1df8b400-b1b4-11eb-808d-bc6a6b2ae9a2.png)

**Reply by Ngm0505** [reply]
Hi Thomas,

Thank you for your reply. However, I don't have "pattern settings" component. May be I need to update my HAL.
That's very kind, thanks

**Comment by Ngm0505** [comment]
Greetings,

Does anybody knows how to display the "pattern settings" component? I have installed the package and reinstalled HAL but I am still getting the image below when I search for "pattern settings".
Thanks for your collaboration

![Picture1](https://user-images.githubusercontent.com/67653786/117729116-aeea8200-b1ea-11eb-9299-a8d4039772c8.png)

**Reply by Thomas-Delaplanche** [reply]
Hi Natago, 
We have updated the packages. I invite you to download the version 1.3.6 here: https://github.com/HALRobotics/Beta/discussions/93#discussioncomment-723423

**Reply by Ngm0505** [reply]
Hi Thomas,
That's very kind.
Thanks to the Team.

**Reply by RZ-021** [reply]
Hi Thomas & team,
 
May I know if the slicer Alpha packages still works? I have tried to download them, but unable to get the PatternSettings component. 
I have both HAL 5 and Hal Robotics Framework installed inside Grasshopper. I also suspect that I have taken the wrong steps when uploading the slicer package to Grasshopper. Hope someone can give some assistance/guide for me. Thanks!

Best regards
Ridhwan
